drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,69 +1,114 @@
-function DragAndDropBlock(runtime, element) {
-
+function DragAndDropBlock(runtime, element, configuration) {
+    "use strict";
+    // Ensure "undefined" has not been redefined (though this unlikely and often impossible).
```

Comment:
@itsjeyd They don't, and it almost never happens in progress. In Chrome and Firefox, it's actually impossible to globally change `undefined`, and even if you create a variable called `undefined` (which is allowed), those browsers will not include that variable whenever you start a new `function` scope. So this is probably pointless and I might just take it out. But in theory, it's possible that in some browsers, someone defined a `var undefined` in a function that wraps this one, and that would lead to bugs (because `undefined` is a global but not a reserved word in JavaScript).

That's the main reason people use the ugly `if (typeof blah.blah === "undefined")` rather than just `if (blah.blah === undefined)`.